src/fragments/lib/auth/common/mfa/flows.mdx

```diff
@@ -0,0 +1,214 @@
+The Auth category supports Multi-factor Authentication (MFA) for user sign-in flows. MFA is an extra layer of security used to make sure that people trying to gain access to an online account are who they say they are. It requires users to provide additional information to verify their identity. The category supports the following MFA methods:
```

Comment:
Small nits:

The Auth category supports Multi-factor Authentication (MFA) for user sign-in flows. MFA is an extra layer of security used to make sure that users trying to gain access to an account are who they say they are. It requires users to provide additional information to verify their identity. The category supports the following MFA methods: